src/form/stories/FormContainer.js

```diff
@@ -45,10 +53,27 @@ class FormContainer extends Component<Props, State> {
     submitAction({ entityData, associationData });
   }
 
+  updateItemIndicies = ({ formData } :Object) => {
+    const pageSection = getPageSectionKey(1, 1);
+    const indexKey = getEntityAddressKey(-1, TASK_ESN, INDEX_FQN);
+    const taskItems = get(formData, getPageSectionKey(1, 1));
```

Comment:
Using `get()` will work on plain objects as well as `Map`.

Reply:
formData is a plain JS object. I am using immutable `get`/`getIn` in case i set formData to an immutable object or if the property i'm trying to access is deeply nested.

Reply:
oh oops, that makes sense :)